src/Mutant.ts

```diff
@@ -68,93 +31,43 @@ export default class Mutant {
     this.scopedTestsById[index] = runResult;
   }
 
-  get columnNumber(): number {
-    return this.mutatedLocation.start.column + 1; //esprima starts at column 0
-  };
-
-  get filename(): string {
-    return this._filename;
-  };
-
-  get lineNumber(): number {
-    return this.mutatedLocation.start.line;
-  };
-
-  get mutatedCode(): string {
-    return this._mutatedCode;
-  };
-
-  get mutatedFilename(): string {
-    return this._mutatedFilename;
-  };
-
-  //TODO: Give this a new name. This string can be longer than a single line. Same goes for 'originalLine'.
-  get mutatedLine(): string {
-    return this._mutatedLine;
-  };
-
-  get mutator(): Mutator {
-    return this._mutator;
-  };
-
-  get originalLine(): string {
-    return this._originalLine;
-  };
-
   /**
-   * @param mutator - The Mutator which was applied to this Mutant.
+   * @param mutatorName - The name of the Mutator which created this mutant.
    * @param filename - The name of the file which was mutated, including the path.
    * @param originalCode - The original content of the file which has not been mutated.
-   * @param substitude - The mutated code which will replace a part of the originalCode.
-   * @param mutatedLocation - The part of the originalCode which has been mutated.
+   * @param replacement - The mutated code which will replace a part of the originalCode.
+   * @param location - The location of the code to be replaced
    */
-  constructor(private _mutator: Mutator, private _filename: string, private originalCode: string, substitude: string, private mutatedLocation: ESTree.SourceLocation) {
-    this.status = MutantStatus.UNTESTED;
-    this.insertSubstitude(substitude);
+  constructor(public mutatorName: string, public fileName: string, private originalCode: string, public replacement: string, public location: ESTree.SourceLocation) {
```

Comment:
https://en.wikipedia.org/wiki/Filename

I guess we can choose. Ok, i'll rename all to `filename`